### ISSUE TITLE
JVM shared library does not contain JNI_Create JavaVM symbol

### ISSUE THREAD

**Issue by srodriguez** [sarl/sarl#1114]
**Describe the bug**
SARL IDE fails to start with the message "The JVM shared library ... does not contain the JNI_Create JavaVM symbol."

**To Reproduce**
1. Launch SARL IDE 0.12

**Screenshots**

![Screenshot 2023-09-12 at 11 05 22 am](https://github.com/sarl/sarl/assets/603295/0b57c383-44d3-4b41-9392-60c9a71e4c6e)

**System configuration:**
-- SARL version: 0.12
-- SARL compiler:	
-  - [ ] Eclipse compiler without Maven	
-  - [ ] Eclipse compiler with Maven	
-  - [ ] Maven compiler on the command line	
-  - [ ] sarlc compiler	
-- Java JDK version (with the manufacturer name):	
openjdk version "1.8.0_382"
OpenJDK Runtime Environment (Temurin)(build 1.8.0_382-b05)
OpenJDK 64-Bit Server VM (Temurin)(build 25.382-b05, mixed mode)

-- Operating System:	
-  - [ ] Linux 64bits	
-  - [ ] Windows 64bits	
-  - [ ] MacOS 64bits	
-  - [X] MacOS M1

**Comment by srodriguez** [comment]
Solution requires to update two files:

**1. eclipse-sarl.ini (in SARLIDE.app/Contents/Eclipse/eclipse-sarl.ini)**

Add -vm paramer as in FAQ (http://www.sarl.io/docs/official/faq/GeneralFAQ.html#25-why-does-sarl-eclipse-fail-on-macos-x).

The final ini file should look similar to:
```
-startup
../Eclipse/plugins/org.eclipse.equinox.launcher_1.5.700.v20200207-2156.jar
--launcher.library
../Eclipse/plugins/org.eclipse.equinox.launcher.cocoa.macosx.x86_64_1.1.1200.v20200508-1552
-vm
/Library/Java/JavaVirtualMachines/temurin-8.jdk/Contents/Home/bin
-vmargs
-Xms256m
-Xmx1g
-XstartOnFirstThread
-Dorg.eclipse.swt.internal.carbon.smallFonts
```

**2. Info.plist (in SARLIDE.app/Contents/Eclipse/Eclipse.app/Contents)**

Add the `<string>-vm</string><string>/Library/Java/JavaVirtualMachines/temurin-8.jdk/Contents/Home/bin/java</string>` inside the `<array>...</array>` section

The final file should look similar to:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
	<dict>
		<key>CFBundleExecutable</key>
		<string>eclipse</string>
		<key>CFBundleGetInfoString</key>
		<string>Eclipse 4.16 for Mac OS X, Copyright IBM Corp. and others 2002, 2020. All rights
			reserved.</string>
		<key>CFBundleIconFile</key>
		<string>Eclipse.icns</string>
		<key>CFBundleIdentifier</key>
		<string>org.eclipse.Eclipse</string>
		<key>CFBundleInfoDictionaryVersion</key>
		<string>6.0</string>
		<key>CFBundleName</key>
		<string>Eclipse</string>
		<key>CFBundlePackageType</key>
		<string>APPL</string>
		<key>CFBundleShortVersionString</key>
		<string>4.16</string>
		<key>CFBundleSignature</key>
		<string>????</string>
		<key>CFBundleVersion</key>
		<string>4.16</string>
		<key>NSHighResolutionCapable</key>
		<true />
		<key>CFBundleDevelopmentRegion</key>
		<string>English</string>
		<key>Eclipse</key>
		<array>
			<string>-vm</string>
			<string>/Library/Java/JavaVirtualMachines/temurin-8.jdk/Contents/Home/bin/java</string>
			<string>-keyring</string>
			<string>~/.eclipse_keyring</string>
		</array>
	</dict>
</plist>
```

**Comment by gallandarakhneorg** [comment]
The provided fix should also b included in the new 0.13.

**Comment by gallandarakhneorg** [comment]
The product 0.13 was successfully launch on a MacOS High Sierra 10.13 with the following Java JDK installed:
```
 java version "17.0.8" 2023-07-18 LTS
 Java(TM) SE Runtme Environment build 17.0.8+9-LTS-211)
 Java HotSpot(TM) 64-Bit Server VM (build 17.0.8+9-LTS-211, mixed mode, sharing)
```

For launch the SARL Eclipse 0.13 on Mac-OS, there is no need to change the `Info.plist` and `eclipse-sarl.ini` files.
